### PR TITLE
polish(xjx): polish pendulum config

### PR DESCRIPTION
**Pull request by sailxjx**
## Check List

- [ ] merge the latest version source branch/repo, and resolve all the conflicts
- [ ] pass style check
- [ ] pass all the tests

### CODE REVIEW

File: dizoo/classic_control/pendulum/config/pendulum_cql_config.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from easydict import EasyDict
 
 pendulum_cql_default_config = dict(
+    exp_name='pendulum_cql_seed0',
     seed=0,
```
Comment (PaParaZz1) [inline dizoo/classic_control/pendulum/config/pendulum_cql_config.py:5]: remove seed in all the config

File: dizoo/classic_control/pendulum/config/pendulum_d4pg_config.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
 from easydict import EasyDict
```
Comment (PaParaZz1) [inline dizoo/classic_control/pendulum/config/pendulum_d4pg_config.py:1]: add runnable entry like example mentioned before

File: dizoo/classic_control/pendulum/config/pendulum_ppo_config.py [hunk 3]
```diff
@@ -23,6 +25,7 @@
             bound_type='tanh',
         ),
         learn=dict(
+            update_per_collect=1,
```
Comment (PaParaZz1) [inline dizoo/classic_control/pendulum/config/pendulum_ppo_config.py:28]: onpolicy algorithm doesn't need `update_per_collect`

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: dizoo/classic_control/pendulum/config/pendulum_cql_config.py (hunk 1 above)
Comment (sailxjx) [reply dizoo/classic_control/pendulum/config/pendulum_cql_config.py]: done

File: dizoo/classic_control/pendulum/config/pendulum_d4pg_config.py (hunk 2 above)
Comment (sailxjx) [reply dizoo/classic_control/pendulum/config/pendulum_d4pg_config.py:1]: done

File: dizoo/classic_control/pendulum/config/pendulum_ppo_config.py (hunk 3 above)
Comment (sailxjx) [reply dizoo/classic_control/pendulum/config/pendulum_ppo_config.py]: done

**Pull request merged**